third_party/protobuf.10007.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/build_defs/BUILD.bazel b/build_defs/BUILD.bazel
```

Comment:
We're trying to remove all existing patch files making it harder to upgrade the dependency. Is it feasible to remove it with help of Protobuf team?

Reply:
This patch is pulled from https://github.com/protocolbuffers/protobuf/issues/10007. The code is in protobuf master branch now, we can delete this patch after the next protobuf release.

Reply:
Great then we can keep this temporarily as we're about to upgrade protobuf to v27 soon.